tox.ini

```diff
@@ -1,14 +1,14 @@
 [tox]
 install_command = pip install {opts} {packages}
 downloadcache = {toxworkdir}/_download/
-envlist = {py36,py37}-{1.11,2.0,2.1,2.2}
+envlist = {py36}-{1.11,2.0,2.1,2.2}
 indexserver =
     default = https://pypi.python.org/simple
 
 [testenv]
 usedevelop = True
 commands =
-  coverage run -p --omit="*tests*" --source=baya --branch \
```

Comment:
This test run is pretty short: no need for a parallel run, and it allows us to get rid of `coverage combine`.